sr_robot_commander/test/test_state_saver_no_hardware.test

```diff
@@ -0,0 +1,3 @@
+<launch>
```

Comment:
for unit test please use rosunit as here https://github.com/shadow-robot/build-servers-check/blob/kinetic-devel/sr_sample_python_library/test/test_sample_python_class.py